src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -0,0 +1,6 @@
+package com.manardenza.utils;
+
+public class DatabaseSeeder {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2094)